model/clusters_mgmt/v1/load_balancer_flavor_type.model

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2023 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Type of node received via telemetry.
+enum LoadBalancerFlavor {
+	// Classic Load Balancer.
+	Classic
+
+	// Network Load Balancer.
+	Nlb
```

Comment:
Should this be NLB or is it ok to keep it as Nlb?

Reply:
I thought about making it NLB but wanted to check of the model processing functionality will then convert it to something like "n_l_b" rather than "nlb" in the JSON which is what we want. I can do either .. just need to double check

Reply:
Oh I see, you can generate a local sdk with these changes by `make generate model_version="HEAD" model_url="<path_to_local_api_model_folder>"`